openqabot/aggrsync.py

```diff
@@ -75,7 +75,10 @@ def __call__(self) -> int:
                 results.append(r)
 
         for r in results:
-            logger.info("Posting aggregate job result to dashboard: %s" % pformat(r))
+            logger.info(
+                "Posting results of %s with status %" % (r["job_id"], r["status"])
```

Comment:
Now the string is the same as in incsyncres. How about

```suggestion
                "Posting aggregate results of %s with status %" % (r["job_id"], r["status"])
```